yt/loaders.py

```diff
@@ -12,6 +12,7 @@
 from multiprocessing import Pipe, Process
 from multiprocessing.connection import Connection
 from pathlib import Path
+from pkg_resources import iter_entry_points
```

Comment:
this import is very expensive (about twice as much as `import numpy`). 
It inflicts a ~10% penalty in the current state of things on `import yt` (which I've been trying very hard to accelerate).
Unless there a cheaper alternative, I suggest moving this statement to `def load`.

Reply:
moved